clients/mod/assign/feedback/onlinejudge/lang/zh_cn/assignfeedback_onlinejudge.php

```diff
@@ -148,4 +148,15 @@
 $string['typeonlinejudge'] = '在线评测';
 $string['usefile'] = '测试用例来自文件';
 $string['waitingforjudge'] = '等待评测结果';
+$string['enabled_help'] = 'Allowing submissions to be programmatically judged';
```

Comment:
Please remove the new English strings from zh-cn and pt-br lang file. Moodle will use English string directly if meet missing string in other languages.

Reply:
Done.